src/vue/pages/TradeExchange/Trade.Orders.vue

```diff
@@ -0,0 +1,149 @@
+              </th>
+              <th>
+                <!-- eslint-disable-next-line -->
+                {{ 'trade-orders.table-order-lbl' | globalize({ asset: assets.quote }) }}
+              </th>
+              <th>
+                <!-- eslint-disable-next-line -->
+                {{ 'trade-orders.table-price-lbl' | globalize({ asset: assets.quote }) }}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr
+              v-for="(order, o) in ordersList"
+              :key="`trade-orders-row-${o}`"
+              @click="selectOrder(order)"
+              :disabled="order.ownerId === accountId">
+              <td>{{ order.baseAmount | formatMoney }}</td>
+              <td>{{ order.quoteAmount | formatMoney }}</td>
+              <td>{{ order.price | formatMoney }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </template>
+    <template v-else-if="isLoading">
+      <loader :message-id="'trade-orders.loading-msg'" />
+    </template>
+    <template v-else>
+      <no-data-message
+        :title-id="'trade-orders.no-data-title'"
+        :message-id="noDataMessage.messageId"
+        :message-id-args="noDataMessage.messageIdArgs"
+      />
+    </template>
+
+    <drawer :is-shown.sync="isSubmitOrderDrawerShown">
+      <template slot="heading">
+        <template v-if="isBuy">
+          {{ 'trade-orders.submit-ask-order-title' | globalize }}
+        </template>
+        <template v-else>
+          {{ 'trade-orders.submit-bid-order-title' | globalize }}
+        </template>
+      </template>
+      <submit-trade-order-form
+        :is-buy="isBuy"
+        :assets="assets"
+        :order="selectedOrder"
+        @close-drawer="closeDrawer"
+      />
+    </drawer>
+  </div>
+</template>
+
+<script>
+import SubmitTradeOrderForm from '@/vue/forms/SubmitTradeOrderForm'
+import NoDataMessage from '@/vue/common/NoDataMessage'
+import Loader from '@/vue/common/Loader'
+import FormMixin from '@/vue/mixins/form.mixin'
+import Drawer from '@/vue/common/Drawer'
+import { vuexTypes } from '@/vuex'
+import { mapGetters } from 'vuex'
+
+const EVENTS = {
+  reloadTrades: 'reload-trades',
+}
+
+export default {
+  name: 'trade-orders',
+  components: {
+    NoDataMessage,
+    Loader,
+    Drawer,
+    SubmitTradeOrderForm,
+  },
+  mixins: [FormMixin],
+  props: {
+    assets: {
+      type: Object,
+      required: true,
+      default: () => ({ base: '', quote: '' }),
+    },
+    isLoading: { type: Boolean, required: true },
+    isBuy: { type: Boolean, required: true },
+    ordersList: { type: Array, required: true },
+  },
+  data: () => ({
+    isSubmitOrderDrawerShown: false,
+    selectedOrder: {},
+  }),
+  computed: {
+    ...mapGetters([
+      vuexTypes.accountId,
+    ]),
+    noDataMessage () {
+      const messageId = this.isBuy
+        ? 'trade-orders.no-data-for-asks-message'
+        : 'trade-orders.no-data-for-bids-message'
```

Comment:
let’s do this within the template

Reply:
I specifically did so to achieve greater purity of the template. Why do I need to move this into the template?

Reply:
Ok let’s leave it as is